homeassistant/components/google_generative_ai_conversation/strings.json

```diff
@@ -18,7 +18,9 @@
     "step": {
       "init": {
         "data": {
+          "recommended": "Recommended settings",
           "prompt": "Prompt Template",
+          "extra_prompt": "Extra prompt template",
```

Comment:
Make it more clear that this will be appended to the default recommended prompt that includes areas and devices?